subprojects/architecture-test/build.gradle.kts

```diff
@@ -29,6 +30,8 @@ tasks.test {
         project.file("src/changes/archunit_store"),
         providers.gradleProperty("archunitRefreeze").map { true })
     )
+
+    useJUnitPlatform()
```

Comment:
Why is this necessary? Isn't it configured in `gradlebuild.unittest-and-compile.gradle.kts`?

Reply:
You are right, removed!